01 - Navigation/01 - Navigate to something you know/1.1. Project Tool Window.txt

```diff
@@ -1,10 +1,10 @@
 Project Tool Window
 
-Shows all files in project.
+Show all files in project.
```

Comment:
I think this should be "Shows" as it describes what the "Project Tool Window" does.

Read it as "(The) Project Tool Window .. *shows* all files in the project"